### ISSUE TITLE
TransformError: Invalid content for node paragraph

### ISSUE THREAD

**Issue by tedchou12** [MO-Movia/licit#28]
```
    {
                "content": [
                    {
                        "content": [
                            {
                                "text": " ",
                                "type": "text"
                            }
                        ],
                        "type": "paragraph"
                    }
                ],
                "type": "paragraph"
            },
            {
                "content": [
                    {
                        "text": "Solution",
                        "type": "text"
                    },
                    {
                        "type": "paragraph"
                    }
                ],
                "type": "paragraph"
            },
```
I think if you have content within paragraph node, it will case dom parse error:
`licit.bundle.js:2 TransformError: Invalid content for node paragraph`

Since I wish to load some html files into editor, if this cannot solve, it cannot be a reliable feature.

**Comment by tedchou12** [comment]
This is the copy pasting of the same word document:
<img width="529" alt="Screen Shot 2020-10-09 at 14 48 28" src="https://user-images.githubusercontent.com/10494709/95547728-904abf80-0a3e-11eb-96e3-396175d4b7c3.png">

**Comment by MO-Movia** [comment]
@tedchou12 , is this an issue?  We have been bringing in extremely large HTML documents (exports from framemaker to HTML documents that are 100's of pages in length), and have not had a problem with the HTML parsing. Have you found something that doesn't paste correctly?

**Comment by tedchou12** [comment]
Thanks.

I am using the HTML to Prosemirror package of PHP, I think the schema was a bit different.

For anyone trying to convert HTML to JSON.

Since this licit editor version has been significantly change from the original prosemirror schema, you might have to use this schema to parse. As for running on the server side, you will have to use jsdom 
https://github.com/tedchou12/prosemirror_server/blob/master/prosemirror_parser/main.js

What I did was use pkg to package the node js as executable, and run in on my linux server.

I figured out dealing with the schema will take too much time, and rewriting the code is not worth it.
